src/test/java/Homework16.java

```diff
@@ -0,0 +1,34 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.safari.SafariDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+import java.time.Duration;
+import java.util.UUID;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework16 extends BaseTest {
+    @Test
+    public static void RegistrationNavigation() throws InterruptedException {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void registrationNavigation()`

![image](https://user-images.githubusercontent.com/104761482/209428163-6311d74d-be0e-481c-8572-f70201317e43.png)